fuse/readonly/readonly_unix.go

```diff
@@ -181,8 +181,11 @@ func (s *Node) Read(req *fuse.ReadRequest, resp *fuse.ReadResponse, intr fs.Intr
 	if err != nil {
 		return err
 	}
-	n, err := io.ReadFull(r, resp.Data[:req.Size])
+	n, err := r.Read(resp.Data[:req.Size])
```

Comment:
I'm not sure if fuse asks for more data if `r.Read()` returns less then `req.Size`.

How about this instead?
```go
var buf bytes.Buffer
n, err := io.Copy(&buf, io.LimitReader(r, req.Size))
if err != nil {
	return err
}
```

`io.Copy` expectes to reach `EOF` and wont return it as an error. The `LimitReader` makes sure we won't return too much data for the fuse request.